table_transforms/simple_join/simple_join.yaml

```diff
@@ -4,20 +4,21 @@ description: |
 
 arguments:
   join_table:
-    type: source
-    description: source ID of the table to join with the source_table
+    type: dataset
```

Comment:
We do not want to restrict joins to only working on datasets rather than operation tables.  Shouldn't this be a table?

Reply:
just made the change to 'table'